cartographer/io/image_utils.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CARTOGRAPHER_IO_IMAGE_UTILS_H_
+#define CARTOGRAPHER_IO_IMAGE_UTILS_H_
+
+#include <cstdint>
+#include <vector>
+
+namespace cartographer {
+namespace io {
+
+class FileWriter;
```

Comment:
do not forward declare: https://google.github.io/styleguide/cppguide.html#Forward_Declarations